src/modules/rlm_imap/Makefile.clean

```diff
@@ -0,0 +1,11 @@
+TARGET		= rlm_example
```

Comment:
You can just delete this whole file.

```
git rm src/modules/rlm_imap/Makefile.clean
git commit --amend .
...
git push --force <branch name>
```

and the github pull request will be automatically updated.